native/membrane_videocompositor_wgpu/src/compositor.rs

```diff
@@ -225,11 +288,11 @@ impl State {
                 label: Some("encoder"),
             });
 
-        for plane in [YUVPlane::Y, YUVPlane::U, YUVPlane::V] {
+        {
             let mut render_pass = encoder.begin_render_pass(&wgpu::RenderPassDescriptor {
                 label: Some("i dont know yet"),
                 color_attachments: &[Some(wgpu::RenderPassColorAttachment {
-                    view: &self.output_textures[plane].view,
+                    view: &self.output_textures.rgba_texture.texture.view,
                     ops: wgpu::Operations {
                         load: wgpu::LoadOp::Clear(wgpu::Color::BLACK),
                         store: true,
```

Comment:
This from the last PR, but maybe put some TODO here to not forget it

Reply:
I don't really know what you mean